cmd/commands/integrations.go

```diff
@@ -474,7 +493,7 @@ func verifyOutputFormat(format string, allowedFormats ...string) error {
 	return fmt.Errorf("invalid output format: %s", format)
 }
 
-func parseGitProvider(provider string) (model.GitProviders, error) {
+func ParseGitProvider(provider string) (model.GitProviders, error) {
```

Comment:
why make this func public? it's only being used inside this package

Reply:
I was using it to verify that the username would pass for bitbucket only